src/main/java/org/threadly/litesockets/NoThreadSocketExecuter.java

```diff
@@ -290,7 +334,7 @@ private void doRead(SelectableChannel sc) {
           readByteBuffer.position(origPos+read);
           resultBuffer.limit(read);
           client.addReadBuffer(resultBuffer.asReadOnlyBuffer());
-          client.callReader();
+          //client.callReader();
```

Comment:
commented out code?  Should this just be removed?  If not, at least add a comment about when you would want to uncomment it.